swagger/api.yaml

```diff
@@ -68,6 +68,9 @@ paths:
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise/260055b/api.yaml#/endpoints/v1/enterpriseCustomerCourseEnrollments"
   "/enterprise/v2/enterprise-customer/{uuid}/learner-summary":
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise-data/8fc95e4/api.yaml#/endpoints/v1/enterpriseCustomerLearnerSummary"
```

Comment:
**Question to reviewers:** Do I also need to update the commit hash for other `edx-enterprise-data` endpoints?

Reply:
Good question - I don't believe its required but I would anyways just to keep them all consistent